apps/condo/domains/common/utils/codegeneration/generate.hooks.ts

```diff
@@ -165,6 +174,39 @@ export function generateReactHooks<
         }, [rowAction, onComplete])
     }
 
+    function useCount (variables: QueryVariables, options?: QueryHookOptions<IUseCountQueryReturnType, QueryVariables>) {
```

Comment:
Why it’s not the same as useObjects types

Reply:
Because different return types, `useCount` does not return objs